commons/src/main/res/values/dimens.xml

```diff
@@ -48,4 +54,10 @@
     <dimen name="big_text_size">18sp</dimen>
     <dimen name="actionbar_text_size">20sp</dimen>
     <dimen name="extra_big_text_size">22sp</dimen>
+
+    <dimen name="cab_popup_menu_vertical_padding">8dp</dimen>
+    <dimen name="cab_popup_menu_max_width">336dp</dimen>
+    <dimen name="cab_popup_menu_min_width">168dp</dimen>
+    <dimen name="cab_popup_menu_width_unit">56dp</dimen>
+    <dimen name="cab_item_min_width">85dp</dimen>
```

Comment:
these numbers are quite arbitrary, do we really need them? Menu should occupy whole width